Updates dependencies

lependu opened the pull request:
May related to #32 which I can't reproduce.

delvedor commented on .travis.yml:
```diff
@@ -1,6 +1,7 @@
 
 node_js:
+  - "10"
   - "9"
```
Can you remove Node v9?

delvedor commented:
@armandopadilla can you give it a try with this branch?

lependu commented:
Sry, I totally forgot about this. :flushed: 
- Updated the example with serving next assets, which might be the missing part for others.
- Also added some npm scripts which may help.
- Created a working example [branch](https://github.com/lependu/fastify-react/tree/example-next6) with `nex/link` and the use case from #27.

mcollina commented on example.js:
```diff
@@ -5,6 +5,14 @@ const fastify = require('fastify')()
+        .then(() => {
+          reply.sent = true
+        })
+    })
```
this block does not really look nice. Is it needed? should we just include this in the module?

lependu replied:
You right it was ugly and unnecessary.
Implemented the static asset handling in the plugin, so I removed the whole example code.
Some side note / question:
- Added an `onClose` hook for closing the next app.
- Also added tests for the implementation, but please note, that we currently testing the next app build in `NODE_ENV=production` only. If you try to run the tests in other NODE_ENV, then it will fail, because the asset map is different dependending on the environment.
- Should we test the asset handling in develoment environment too? If I not mistaken, the he hard part is that we build the next app before the tests, so if we want to add tests in other environment, than we need to add a different npm script eg `test:dev` and a different test file eg `test-development.js`
